x-pack/plugins/apm/public/store/reactReduxRequest/license.tsx

```diff
@@ -4,26 +4,31 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React from 'react';
+import { Request, RRRRender } from 'react-redux-request';
+import { LicenseApiResponse, loadLicense } from '../../services/rest/xpack';
+import { IReduxState } from '../rootReducer';
 import { createInitialDataSelector } from './helpers';
-import { Request } from 'react-redux-request';
-import { loadLicense } from '../../services/rest/xpack';
 
 const ID = 'license';
 const INITIAL_DATA = {
   features: {
-    watcher: { isAvailable: false },
+    watcher: { is_available: false },
     ml: { isAvailable: false }
   },
-  license: { isActive: false }
+  license: { is_active: false }
```

Comment:
I think `callApi` still camelCases response objects automatically, so not sure this is right.
If that's the case you can disable camecasing for just the license endpoint. In a follow-up PR I want to disable it by default, and only have i camelcase the places that need it. And they should eventually be migrated to not use the camelcased props.